### PR TITLE
PXC-3340: merge codership/mysql-wsrep 5.6.48

### PR DESCRIPTION
**Pull request by kamil-holubicki**
Merge codership/mysql-wsrep tag 'wsrep_5.6.48-25.30'

### CODE REVIEW

**Comment by it-percona** [comment]
[CLA assistant check](https://cla-assistant.percona.com:443/percona/percona-xtradb-cluster?pullRequest=1176) <br/>Thank you for your submission! We really appreciate it. Like many open source projects, we ask that you all sign our [Contributor License Agreement](https://cla-assistant.percona.com:443/percona/percona-xtradb-cluster?pullRequest=1176) before we can accept your contribution.<br/>**1** out of **5** committers have signed the CLA.<br/><br/>:white_check_mark: kamil-holubicki<br/>:x: temeo<br/>:x: sjaakola<br/>:x: mkaruza<br/>:x: ayurchen<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.percona.com:443/check/percona/percona-xtradb-cluster?pullRequest=1176) it.</sub>

File: mysql-test/mysql-test-run.pl [hunk 1]
```diff
@@ -2359,28 +2359,37 @@ ($)
...
+	$ENV{$plug_var.'_LOAD_ADD_PATH'}= $load_add_var_with_path;
       }
     } else {
       $ENV{$plug_var}= "";
       $ENV{$plug_var.'_DIR'}= "";
       $ENV{$plug_var.'_OPT'}= "";
       $ENV{$plug_var.'_LOAD'}= "" if $plug_names;
       $ENV{$plug_var.'_LOAD_ADD'}= "" if $plug_names;
+      $ENV{$plug_var.'_LOAD_PATH'}= "" if $plug_names;
+      $ENV{$plug_var.'_LOAD_ADD_PATH'}= "" if $plug_names;
```
Comment (dutow) [inline mysql-test/mysql-test-run.pl:2392]: Same question as with galera: how is this related to the conflict resolution?

File: mysql-test/suite/galera/t/galera_migrate.test [hunk 2]
```diff
@@ -226,32 +208,18 @@ DROP TABLE t1;
 --connection node_2
 DROP TABLE t1;
 
---connection node_4
+--connection node_3
 SET GLOBAL wsrep_provider = 'none';
-SET GLOBAL wsrep_sst_method = 'rsync';
```
Comment (dutow) [inline mysql-test/suite/galera/t/galera_migrate.test]: PXC 5.6 doesn't support the rsync sst method?

Comment (kamil-holubicki) [reply mysql-test/suite/galera/t/galera_migrate.test]: fixed

**Review by dutow — APPROVED** [review]
